commit 283f778908873eca61388029fc418fa800c9d7d7
Author: Jia Tan <jiat0218@gmail.com>
Date:   Mon Jan 22 23:33:39 2024 +0800

    xz: Update the man page for the RISC-V Filter.
    
    A special note was added to suggest using four-byte alignment when the
    compressed instruction extension is not present in a RISC-V binary.

diff --git a/src/xz/xz.1 b/src/xz/xz.1
index 2e289ba2..b489fd35 100644
--- a/src/xz/xz.1
+++ b/src/xz/xz.1
@@ -6,7 +6,7 @@
 .\" This file has been put into the public domain.
 .\" You can do whatever you want with this file.
 .\"
-.TH XZ 1 "2024-01-19" "Tukaani" "XZ Utils"
+.TH XZ 1 "2024-01-22" "Tukaani" "XZ Utils"
 .
 .SH NAME
 xz, unxz, xzcat, lzma, unlzma, lzcat \- Compress or decompress .xz and .lzma files
@@ -1857,6 +1857,7 @@ ARM64;4;4096-byte alignment is best
 PowerPC;4;Big endian only
 IA-64;16;Itanium
 SPARC;4;
+RISC-V;2;
 .TE
 .RE
 .RE